upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -103,6 +100,11 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap, secretStore fi.SecretS
 	dest["trimPrefix"] = sprigTxtFuncMap["trimPrefix"]
 	dest["semverCompare"] = sprigTxtFuncMap["semverCompare"]
 	dest["ternary"] = sprigTxtFuncMap["ternary"]
+	dest["dict"] = sprigTxtFuncMap["dict"]
+	dest["list"] = sprigTxtFuncMap["list"]
+	dest["set"] = sprigTxtFuncMap["set"]
+	dest["append"] = sprigTxtFuncMap["append"]
+	dest["join"] = sprigTxtFuncMap["join"]
```

Comment:
It's probably best to do this as a separate PR or at least a separate commit.

Reply:
I'll make a separate PR to use sprig `join`.